source/_integrations/netgear.markdown

```diff
@@ -95,7 +95,7 @@ Displays if the device is currently connected to the router (Home) or not (Away)
 
 Switch that lets you Allow or Block a device on the Network.
 For this entity to actually Block the device, "Access Control" needs to be turned on in the Router settings.
-Enable the `Access Control` switch entity and turn it on.
+Enable the `Access Control` switch entity and turn it on.  To use this feature you must enable the switch for each connected device under Configuration->Advanced Settings on the entity dialog.
```

Comment:
The reason this is added, is because in general it is not clear you can enable extra entities.

A change to the general UX in Home Assistant has been made to make that clear.

I'm going to close this PR for that reason, as any integration can provide entities that are disabled by default (which is not specific to this integration).

../Frenck